cfgov/unprocessed/css/enhancements/tables.less

```diff
@@ -2,6 +2,24 @@
   name: Tables
   family: cf-tables
   notes:
+    - A fix that makes stacking cells 100% width when stacked
+  tags:
+    - cfgov-cf-tables
+*/
+
+.respond-to-max(@bp-xs-max, {
+    .o-table__stack-on-small {
```

Comment:
Here was my thinking: The enhancements files seem to contain fixes addressed towards specific issues. I figured I _could_ put this fix inside of that snippet, but it seemed like that snippet addressed a different issue, and I thought combining them might be a poor plan. My next step is to take this fix to the `canary` branch of `capital-framework`, so theoretically, I can remove this little fix from `/enhancements/tables.less` in the future when `canary` comes to `cfgov-refresh`.

But, all that said, I'm happy to defer to your judgment on this! 😄